impact.tex

```diff
@@ -57,7 +54,24 @@ \subsection{Pathway toward impact}
 \end{draft}
 
 
-\TOWRITE{}{Start with brief narrative summary, hitting 'Scale and Significance' hard. We have close to nothing on Significance, yet. Scale could be clearer: All researchers!}
+Almost all researchers perform some kind of computation in the course of their work,
+from large-scale simulations to production of simple charts and figures.
```

Comment:
I find it weird to have large-scale first and then simple charts. I would have written:

from the production of simple charts and figures to large-scale simulations.

Reply:
That makes sense. I was going for expected (big simulations, obviously reproducibility is relevant!)->unexpected (wait, reproducibility even applies to my excel charts?!), but small->big is better.

Reply:
Interesting. I was wondering why you had "big" first and then small!
My thinking is opposite because small is easier to reproduce but large is not. I often hear that it is OK not to be reproducible on HPC because it is complex and so costly that you will never tried to reproduce anything...